Example/Swift/ViewController.swift

```diff
@@ -132,9 +132,8 @@ class ViewController: UIViewController, MGLMapViewDelegate, AVSpeechSynthesizerD
              Otherwise, it will continue to compare the user to the old route and continually reroute the user.
              */
             self.navigation?.routeProgress = RouteProgress(route: self.userRoute!)
+            self.navigation?.routeProgress.currentLegProgress.stepIndex = 0
```

Comment:
The previous line creates a new route progress, and therefore a new leg progress. Why is the leg progress’s step index anything other than 0 at this point?